controllers/members.js

```diff
@@ -49,7 +53,37 @@ const getIdleMembers = async (req, res) => {
   }
 }
 
+const purgeMembersCache = async (req, res) => {
+  try {
+    const { username } = req.userData?.username
+
+    if (!username) {
+      return res.boom.badRequest('Username is not valid')
+    }
+
+    const rep = await axios.post(
+      CLOUD_FARE_PURGE_CACHE_API,
+      {
+        files: [`https://members.realdevsquad.com/${username}`]
+      },
+      {
+        headers: {
+          'X-Auth-Key': config.get('CLOUD_FARE_X_AUTH_KEY'),
+          'X-Auth-Email': config.get('CLOUD_FARE_AUTH_EMAIL'),
+          Authorization: `Bearer ${config.get('CLOUD_FARE_WORDPRESS_AUTHORIZATION_TOKEN')}`
+        }
+      }
+    )
+
```

Comment:
We already have a wrapper function `fetch` in `utils/fetch.js` for network calls which in turn uses `axios`
Please use that

Reply:
Done